website/docs/configure_hydra/workdir.md

```diff
@@ -36,16 +31,102 @@ Run output directory can contain user configuration variables:
 hydra:
   run:
     dir: outputs/${now:%Y-%m-%d_%H-%M-%S}/opt:${optimizer.type}
+```
+
+### Configuration for multirun
+
+Default multirun dir configurations:
+```yaml
+hydra:
+  sweep:
+    dir: multirun/${now:%Y-%m-%d}/${now:%H-%M-%S}
+    subdir: ${hydra.job.num}
+```
+Run the example application
+```bash
+python my_app.py --multirun a=a1,a2,a3 
+```
+
+would create working dir structure like:
+```bash
+$ tree -d multirun/
+multirun/
+└── 2021-03-10
+    └── 17-21-00
+        ├── 0
+        ├── 1
+        └── 2
+```
+Similar configuration patterns in run can be applied to config multirun dir as well.
+
+For example, multirun output directory grouped by job name, and sub dir by job num:
+```yaml
+hydra:
+  sweep:
+    dir: ${hydra.job.name}
+    subdir: ${hydra.job.num}
+```
+
+### Using `hydra.job.override_dirname`
+
+<ExampleGithubLink text="Example application" to="examples/configure_hydra/job_override_dirname"/>
+
+This field is populated automatically using your command line arguments and is typically being used as a part of your 
+output directory pattern. It is meant to be used along with the configuration for working dir, especially
+in `hydra.sweep.subdir`.
+
+For example, we configure the example application like the following
+```yaml
+hydra:
+  sweep:
+    dir: multirun
+    subdir: ${hydra.job.override_dirname}
+```
+
+then run the application with command-line overrides:
 
+```bash
+python my_app.py --multirun batch_size=32 learning_rate=0.1,0.01
 ```
+Would result in creating the following output structure
+```bash
+$ tree multirun -d
+multirun
+├── batch_size=32,learning_rate=0.01
+└── batch_size=32,learning_rate=0.1
```

Comment:
If this does not fit in a two column layout next to the corresponding config you can use different key names.

Reply:
Fixed :)